src/oadp_resources/oadp/datamover/volume_snapshot_backup.py

```diff
@@ -53,11 +53,13 @@ def replication_source_completed(self):
             conditions[1].type == "Synchronizing" and \
             not conditions[1].status
 
-    def done(self):
+    def done(self, log_status=False):
```

Comment:
do we really need to flag this?

Reply:
+1 we should log by default

Reply:
Do you think we should log all the status only for this done function by default?
Or for all the other phase functions?

Reply:
I patched it , done.